src/main/java/org/altbeacon/beacon/startup/RegionBootstrap.java

```diff
@@ -96,6 +97,42 @@ public void disable() {
         beaconManager.unbind(beaconConsumer);
     }
 
+    /**
+     * Add a new region
+     *
+     * @param region
+     */
+    public void addRegion(Region region) {
+        if (!regions.contains(region)) {
+            if (serviceConnected) {
```

Comment:
It might be  a good idea to log an error in the else case here -- if the user tries to add a Region when the service is not connected.